app/templates/components/hyper-table/cell-renderers/numeric.hbs

```diff
@@ -1,5 +1,11 @@
-{{#if emptyValue}}
-  —
-{{else}}
-  {{format-numeric value}}
-{{/if}}
+<div class="editing-input" style="">
+  {{#if isEditing}}
+    {{input type="number" focus-out=(action "toggleEditing") type="text" value=value class="editing-input__field form-control"}}
+    <i class="fa fa-save margin-left-xx-sm" onclick={{action "toggleEditing"}}/>
+  {{else}}
+    <span data-toggle="tooltip" data-title={{format-numeric value}}>
```

Comment:
what happen when you pass empty value to `format-numeric` ?

Reply:
There will be no value and `emptyValue` will be `true`, so will switch to the Empty Case